Error: Cannot find module '../src/helpers/icns.json'

There seems to be an issue with the latest release:

```bash
> note-boi@0.1.2 icon
> npx @tauri-apps/tauricon ./artwork/icon.png

node:internal/modules/cjs/loader:933
  const err = new Error(message);
              ^

Error: Cannot find module '../src/helpers/icns.json'
Require stack:
- C:\Users\Dan\AppData\Local\npm-cache\_npx\98c943a920688aff\node_modules\@tauri-apps\tauricon\dist\tauricon.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at file:///C:/Users/Dan/AppData/Local/npm-cache/_npx/98c943a920688aff/node_modules/@tauri-apps/tauricon/dist/tauricon.js:1:3428
    at ModuleJob.run (node:internal/modules/esm/module_job:185:25)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:281:24)
    at async loadESM (node:internal/process/esm_loader:88:5)
    at async handleMainPromise (node:internal/modules/run_main:65:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'C:\\Users\\Dan\\AppData\\Local\\npm-cache\\_npx\\98c943a920688aff\\node_modules\\@tauri-apps\\tauricon\\dist\\tauricon.js'
  ]
}
```

I tried deleting then reinstalling, but that didn't fix the issue.

`tauri info`:

```
Environment
  › OS: Windows 10.0.22000 X64
  › Webview2: 100.0.1185.50
  › MSVC: 
  › Node.js: 16.13.1
  › npm: 8.3.2
  › pnpm: Not installed!
  › yarn: Not installed!
  › rustup: 1.24.3
  › rustc: 1.60.0
  › cargo: 1.60.0
  › Rust toolchain: stable-x86_64-pc-windows-msvc 

Packages
  › @tauri-apps/cli [NPM]: 1.0.0-rc.9
  › @tauri-apps/api [NPM]: 1.0.0-rc.4
  › tauri [RUST]: 1.0.0-rc.8,
  › tauri-build [RUST]: 1.0.0-rc.7,
  › tao [RUST]: 0.8.3,
  › wry [RUST]: 0.15.1,

App
  › build-type: bundle
  › CSP: default-src 'self'
  › distDir: ../dist
  › devPath: http://localhost:3000/
  › framework: Vue.js

App directory structure
  ├─ .git
  ├─ .github
  ├─ .vscode
  ├─ artwork
  ├─ dist
  ├─ node_modules
  ├─ scripts
  ├─ src
  └─ src-tauri
```

--- a/.changes/include-json.md
+++ b/.changes/include-json.md
@@ -1,0 +1,5 @@
+---
+"tauricon": patch
+---
+
+Include icns.json file in generated package.

--- a/package.json
+++ b/package.json
@@ -22,7 +22,8 @@
   "files": [
     "bin",
     "dist",
-    "scripts"
+    "scripts",
+    "src/helpers/icns.json"
   ],
   "scripts": {
     "postinstall": "",

